todo-app/src/TodoList.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import {inject, observer} from 'mobx-react';
+import Todo from './Todo';
+
+@inject('todos')@observer
+export default class TodoList extends React.Component {
+
+  state = {
+    editingTodoID: null,
+  };
+
+  handleEditTodo = (id) => {
+    this.setState({editingTodoID: id});
+  };
+  
+  handleTodoCancelEdit = () => {
+    this.setState({editingTodoID: null})
+  };
+
+  handleTodoSave = (id, text) => {
+    this.props.todos.onSaveTodo(id, text);
```

Comment:
Small thing: I would name it `saveTodo`, as it's not an event handler, but simply a method on the store.